.github/workflows/CI.yml

```diff
@@ -704,6 +705,51 @@ jobs:
           cd testing-framework/terraform
           make terraformCleanup
 
+  run-logs-testbed:
+    runs-on: ubuntu-latest
+    needs: [ e2etest-preparation, create-test-ref]
+
+    steps:
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.COLLECTOR_ASSUMABLE_ROLE_ARN }}
+          aws-region: us-west-2
+          # 6 hours
+          role-duration-seconds: 21600
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
```

Comment:
I meant that you don't need the `Set up QEMU` step  since we are not cross compiling a docker image here.

Reply:
Yes, Actually we don't need it at all. since we already use [docker login action](https://github.com/marketplace/actions/docker-login#aws-elastic-container-registry-ecr). Removed

Reply:
No, docker-login does not have anything todo with setup-qemu.

Reply:
oh yes i meant, docker is already is in place. we dont need the setup. appreciate the call out.